bin/mapbox-gl-style-build

```diff
@@ -44,20 +44,22 @@ const argv = yargs(hideBin(process.argv))
   .parse();
 
 const buildStyles = async (styleDir, layerDir, outputDir, options = {}) => {
-  const verbose = options?.verbose ?? false;
```

Comment:
This wasn't being used here at all since `options` are passed into `buildStyle` wholesale.